utils/doclint/check_public_api/index.js

```diff
@@ -42,6 +43,10 @@ const EXCLUDE_METHODS = new Set([
   'JSHandle.toString',
 ]);
 
+const EXCLUDE_PROPERTIES = new Set([
```

Comment:
Methods are properties. I think you can just rename EXCLUDE_METHODS to EXCLUDE_PROPERTIES and add `TimeoutError.name` into it. They can't conflict.

Reply:
Done.